airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityTest.java

```diff
@@ -196,42 +192,6 @@ void createResetJob() throws JsonValidationException, ConfigNotFoundException, I
       Assertions.assertThat(output.getJobId()).isEqualTo(JOB_ID);
     }
 
-    @Test
-    @DisplayName("Test attempt creation")
-    void createAttempt() throws IOException {
```

Comment:
This test can be removed because there is already one that call the other method.